cdap-source-control/src/main/java/io/cdap/cdap/sourcecontrol/operationrunner/ListAppsContext.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.sourcecontrol.operationrunner;
+
+import io.cdap.cdap.proto.id.NamespaceId;
+import io.cdap.cdap.proto.sourcecontrol.RepositoryConfig;
+
+/**
+ * Information required by {@link SourceControlOperationRunner} to execute the task of
+ * listing application configs in linked repository.
+ */
+public class ListAppsContext {
```

Comment:
nit: would be better to name this something like NamespaceRepository. Context classes are normally in our api modules to be used by app/plugins/extensions. This class also isn't necessarily specific to listing apps.

Reply:
Makes sense. renamed